docs/src/main/sphinx/release/release-406.md

```diff
@@ -0,0 +1,67 @@
+# Release 406 (6 Jan 2023)
+
+## General
+
+* Add support for exchange spooling on HDFS when fault-tolerant execution is
```

Comment:
Lets make sure the doc PR for this is merged and maybe even link to that section here. I kinda think that is what we did when we added the others